deploy/docker/docker-compose.yml

```diff
@@ -1,33 +1,69 @@
 version: '2'
 
 services:
-  node1:
-    image: sshd
+  nstat:
+    image: intracom/nstat
+    privileged: true
+    hostname: nstat-node
```

Comment:
It helps when ssh-ing to a docker-container. It directly shows where you are logged in.